package.json

```diff
@@ -91,6 +91,7 @@
     "lodash.merge": "^4.6.2",
     "lodash.set": "^4.3.2",
     "node-cache": "^5.1.2",
+    "pdf-lib": "1.17.1",
```

Comment:
i supposed you did your benchmark already, but that is surprising no recently maintained library enables PDF manipulation.

Reply:
Yes, the library may change with adding PDF/A3 compliancy though.